tests/reading-tests.md

```diff
@@ -295,68 +294,89 @@ Drawing also from the first unit test, Leila concludes:
 
 To summarize, this second test expects the value of `fizzBuzz(25)` to equal `'Buzz'`.
 
-## Example on Replit.com
+## Example on CodeSandbox.io
+
+We can run a version of this test suite on [this CodeSandbox project](https://codesandbox.io/s/fizzbuzz-with-tests-488sq?file=/test/fizzbuzz.test.js).
+
+CodeSandbox.io is a platform for creating projects. We'll use this platform to read, write, and run our FizzBuzz unit tests.
+
+These projects need to be configured to use the Jest package. Let's purposely skip over those details in order to focus on the tests. The tests in [this CodeSandbox project](https://replit.com/@adacore/FizzBuzz-With-Tests#index.js) are already set up.
```

Comment:
still had a replit link. I just copied the same link as the first codesandbox link. is that OK?
```suggestion
These projects need to be configured to use the Jest package. Let's purposely skip over those details in order to focus on the tests. The tests in [this CodeSandbox project](https://codesandbox.io/s/fizzbuzz-with-tests-488sq?file=/test/fizzbuzz.test.js) are already set up.
```